java-common-artifacts/src/main/java/com/fullcontact/apilib/models/Tag.java

```diff
@@ -0,0 +1,26 @@
+package com.fullcontact.apilib.models;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+@AllArgsConstructor(access = AccessLevel.PROTECTED)
+public class Tag {
+  private String key;
+  private String value;
+
+  public boolean isValid() {
+    if (this.key != null && !this.key.isEmpty() && this.value != null && !this.value.isEmpty()) {
+      return true;
+    }
```

Comment:
Add one more check: Don't allow `'` in the tag key. Reference: https://github.com/fullcontact/nimbus/blob/master/nimbus-service/src/main/java/fc/nimbus/resources/AudienceResource.java#L131

Reply:
Done